pkg/security/probe/applier.go

```diff
@@ -97,7 +97,7 @@ func (rsa *RuleSetApplier) Apply(rs *rules.RuleSet, approvers map[eval.EventType
 			return nil, errors.Wrap(err, "failed to select probes")
 		}
 
-		if err := rsa.probe.FlushDiscarders(); err != nil {
+		if err := rsa.probe.OnRuleSetApplied(); err != nil {
```

Comment:
Should the error message below be updated too?

Reply:
yes, I'll update, thx